### PR TITLE
LPD-38626 [Test fix] ObjectForms#CanSendFormEmailWhenItIsRelatedWithObject

### PR DESCRIPTION
**Pull request by larissacribeiro**
Related Issue: [LPD-38626](https://liferay.atlassian.net/browse/LPD-38626)

[LPD-38626]: https://liferay.atlassian.net/browse/LPD-38626?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by larissacribeiro** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectForms.testcase [hunk 1]
```diff
@@ -613,8 +613,8 @@ definition {
 				fieldValue = "Entry test");
 
 			AssertClick(
-				locator1 = "Button#SUBMIT_FOR_WORKFLOW",
-				value1 = "Submit for Workflow");
+				locator1 = "Button#SAVE",
+				value1 = "Save");
```
Comment (igor-franca) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectForms.testcase:617]: Hey @larissacribeiro, can we relate this change in the UI to any task mapped by our team?

Are we sure this is the new expected behavior?

Comment (larissacribeiro) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectForms.testcase:617]: Hey @igor-franca! yes, it is the new behavior, you can see the related bug [here](https://liferay.atlassian.net/browse/LPD-35127).

This change broke many other tests, which were fixed on [LPD-37725](https://liferay.atlassian.net/browse/LPD-37725), and We just missed fixing this one too.

**Review by igor-franca — APPROVED** [review]
LGTM!

**Comment by larissacribeiro** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**